components/tikv_alloc/src/lib.rs

```diff
@@ -92,27 +92,22 @@
 #[cfg(feature = "mem-profiling")]
 #[macro_use]
 extern crate log;
+pub mod default;
```

Comment:
This shouldn't be pub. If you had a problem with resolution that required it to be pub then maybe `pub(crate)` will work.

Reply:
i see